compiler/src/dmd/backend/debugprint.d

```diff
@@ -328,7 +328,6 @@ void WRdefnod(ref GlobalOptimizer go)
   }
 }
 
-@trusted
```

Comment:
any reason for not marking these `@safe`?

Reply:
There's `@safe:` on top of the file

Reply:
Ha! On-mass attributes here to bite you again!